### DISCUSSION TITLE
Conversation using chatgpt

### DISCUSSION THREAD

**Discussion by ARudiuk** [briansunter/logseq-plugin-gpt3-openai#166] [Ideas]
Thanks for the extension, and sorry if this is the wrong place to post.

I was wondering if it is possible to have a conversation with a GPT model using this extension? I looked through the examples on the plugin page, but none of them seem to show continued conversations. 

An example would be:

Break this block into learning points
-Learning points
Order the learning points based on easiest to hardest to learn
-Ordered learning points

**Comment by juanjonavarro** [comment]
You can create a new page and execute the 'gpt-page' command on it. This command sends the complete page to the OpenAI API, enabling you to have a conversation with GPT by calling this command repeatedly. However, please keep in mind that this method only works if your page is below the token limit of the model (which is 4097 for GPT-3.5).
